src/resources/messages.ts

```diff
@@ -210,7 +210,18 @@ export class Messages extends Resource {
     if (attachmentSize >= Messages.MAXIMUM_JSON_ATTACHMENT_SIZE) {
       requestOptions.form = Messages._buildFormRequest(requestBody);
     } else {
-      requestOptions.body = requestBody;
+      if (requestBody.attachments) {
```

Comment:
As the documentation said:
_`To add attachments to a draft with a total HTTP payload size under 3MB, you can use either the application/json format or the multipart/form-data format.`_

The reduce above only gets the last attachment's size. 
I think the correct reduce would be:
```ts
 const maxLength = attachments.reduce((acc, attachment) => {
   acc += (attachment.size as number) || 0;
     return acc;
    }, 0) || 0;
```

Reply:
🤦🏼 Of course. Thanks for pointing this out, will rectify it 😄